velox/functions/prestosql/tests/GreatestLeastTest.cpp

```diff
@@ -68,6 +72,11 @@ TEST_F(GreatestLeastTest, nanInput) {
         runTest<double>("greatest(c0)", {1, {0.0 / 0.0}}, {1, 0});
       },
       "Invalid argument to greatest(): NaN");
+
+  // Ensure it works with Try.
+  std::vector<double> input{0, 1.1, std::nan("1")};
+  runTest<double>("try(least(c0, 1.0))", {input}, {0, 1.0, std::nullopt});
```

Comment:
It would be easier to read if greatest test cases were next to each other and least test cases were also next to each other. It would also be nice if both failing and succeeding expressions were exactly the same and ran on same input data.